src/server/graphql/subscriptions/organizationUpdated.js

```diff
@@ -0,0 +1,25 @@
+import {GraphQLNonNull} from 'graphql';
+import getRethink from 'server/database/rethinkDriver';
+import makeSubscribeIter from 'server/graphql/makeSubscribeIter';
+import AddOrgPayload from 'server/graphql/types/AddOrgPayload';
+import {getUserId} from 'server/utils/authorization';
+import {ORGANIZATION_UPDATED} from 'universal/utils/constants';
+
+
+export default {
+  type: new GraphQLNonNull(AddOrgPayload),
+  subscribe: async (source, args, {authToken, socketId}) => {
```

Comment:
This is the reason for bumping the GraphQL version. v0.11 allows us to return a promise that ultimately resolves to an AsyncIterator.

Reply:
Well that's interesting!